src/serlio/serlioPlugin.cpp

```diff
@@ -118,3 +104,20 @@ MStatus uninitializePlugin(MObject obj) {
 	}
 	return status;
 }
+
+namespace MayaPluginUtilities {
+
+bool pluginDependencyCheck(const std::vector<std::string>& dependencies) {
+	for (const auto& d : dependencies) {
+		MString md(d.c_str());
+		auto p = MFnPlugin::findPlugin(md);
+		if (p.isNull()) {
+			MGlobal::displayError("Serlio: the required dependency '" + md +
+			                      "' is not loaded, please activate it and restart Maya!");
+			return false;
+		}
+	}
+	return true;
+}

```

Comment:
is there a reason we don't return an `MStatus` directly here?

Reply:
Cannot use maya types in the header because it is included in the unit tests which do not link against the Maya API.

This is a compromise, we should eventually split up the header into a separate one which contains the stuff for the unit tests.